tests/WorkerTest.php

```diff
@@ -164,18 +149,22 @@ public function runNextJob_EntityManagerAndConnectionOpen_ShouldClearEntityManag
     public function runNextJob_JobThrowsException_ShouldRequeueJobAndNotKillWorkerProcess(): void
     {
         // Assemble
-        $this->entityManager->shouldReceive('isOpen')->andReturn(true);
-        $this->entityManager->shouldReceive('clear');
+        $this->entityManager->shouldReceive('getConnection')->andReturn($this->connection)->once();
+        $this->entityManager->shouldReceive('isOpen')->andReturn(true)->once();
+        $this->entityManager->shouldReceive('clear')->once();
+
+        $this->platform->shouldReceive('getDummySelectSQL')->andReturn('SELECT 1')->once();
 
-        $this->connection->shouldReceive('ping')->andReturn(true);
+        $this->connection->shouldReceive('executeQuery')->once();
+        $this->connection->shouldReceive('getDatabasePlatform')->andReturn($this->platform)->once();
         $this->connection->shouldNotReceive('close');
         $this->connection->shouldNotReceive('connect');
 
         $job = Mockery::mock(Job::class);
-        $job->shouldReceive('handle')->andThrow(new Exception('test'));
-        $job->shouldReceive('isDeleted')->andReturn(false);
-        $job->shouldReceive('isReleased')->andReturn(false);
-        $job->shouldReceive('hasFailed')->andReturn(false);
+        $job->shouldReceive('fire')->andThrow(new Exception('test'))->once();
```

Comment:
The method on Laravel's job class changed from `handle()` to `fire()` between 9.x and 10.x then?